content/en-us/luau/metatables.md

```diff
@@ -118,19 +118,19 @@ Here's the list of available metamethods:
 		  <td>__mode</td>
 		  <td>Used in weak tables, declaring whether the keys and/or values of a table are weak. **Note:** References to Roblox instances are never weak. Tables that hold such references will never be garbage collected.</td>
 		</tr>
-	  <tr>
-		  <td>__gc(table)</td>
-		  <td>Fired when the table is garbage-collected. **Note:** On Roblox, this metamethod is disabled.</td>
-		</tr>
 	  <tr>
 		  <td>__len(table)</td>
 		  <td>Fired when the # length operator is used on the Object.</td>
 		</tr>
+	  <tr>
+		  <td>__iter(table)
+		  <td>Used to denote a custom iterator when using generalized iteration.</td>
+		  </td>
```

Comment:
Awesome, thanks for adding this. Agreed that we should add a note to https://create.roblox.com/docs/luau/control-structures#for-loops as well, but that doesn't need to be done in this PR.